### PR TITLE
[NEW] Whatsapp url card integration

### LINKED ISSUE #727: [CX] Verify the whatsApp card format and test sending it from our agent to a whatsApp client through twillio integration.

### PR DESCRIPTION
**Pull request by AlexanderKanakis**

### CODE REVIEW

**Comment by ear-dev** [comment]
@AlexanderKanakis this PR seems to have failed its tests.....  can you take a look please? thanks

**Comment by AlexanderKanakis** [comment]
> @AlexanderKanakis this PR seems to have failed its tests..... can you take a look please? thanks

Fixed ESlint Issues

**Comment by ear-dev** [comment]
@Shailesh351 can you review please?  Thanks!

File: app/livechat/server/sendMessageBySMS.js [hunk 1]
```diff
@@ -36,6 +36,11 @@ callbacks.add('afterSaveMessage', function(message, room) {
 		extraData = Object.assign({}, { rid, userId, fileUpload });
 	}
 
+	if (message.customFields && message.customFields.imageCardURL) {
```
Comment (Shailesh351) [inline app/livechat/server/sendMessageBySMS.js]: @AlexanderKanakis Functionality working great! I was able to get pdf and image as well.

IMO it's better to rename `imageCardURL` to something more generic, as functionality is not limited to Images 😄

Comment (ear-dev) [reply app/livechat/server/sendMessageBySMS.js]: maybe just 'cardURL'?

Comment (ear-dev) [reply app/livechat/server/sendMessageBySMS.js]: mediaCardUrl

**Pull request merged**